src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetDayName.cs

```diff
@@ -47,7 +47,8 @@ public void GetDayName(DateTimeFormatInfo format, string[] expected)
 
         public void GetDayName_Invalid()
         {
-            Assert.Throws<ArgumentOutOfRangeException>(() => new DateTimeFormatInfo().GetDayName((DayOfWeek)(-1))); // DayOfWeek is invalid
+            Assert.Throws<ArgumentOutOfRangeException>("value", () => new DateTimeFormatInfo().GetDayName(DayOfWeek.Sunday - 1)); // DayOfWeek is invalid
+            Assert.Throws<ArgumentOutOfRangeException>("value", () => new DateTimeFormatInfo().GetDayName(DayOfWeek.Saturday + 1)); // DayOfWeek is invalid
```

Comment:
Well my reasoning is that these data are finite: it's unlikely we will add anymore. Therefore using inlinedata adds code bloat.

That's my personal opinion thoug